src/labkey/security/Policy.ts

```diff
@@ -63,17 +48,31 @@ export function deletePolicy(config: DeletePolicyOptions): XMLHttpRequest {
     });
 }
 
+export interface Policy {
+    assignments: {
+        role: string
+        userId: number
+    }[]
+    modified: string
+    modifiedMillis: number
+    resourceId: string
+    requestedResourceId: string
+}
+
 export interface GetPolicyOptions {
```

Comment:
does this one not extend RequestCallbackOptions because of the additional success parameter?

Reply:
Yes, precisely. Changing the signature of the `success` method parameters beyond the type of the initial `data` is not supported by `RequestCallbackOptions`.